lib/Wikia/src/Service/User/Permissions/data/PermissionsDefinesAfterWikiFactory.php

```diff
@@ -641,3 +641,5 @@
 $wgGroupPermissions['*']['fandom-admin'] = false;
 $wgGroupPermissions['staff']['fandom-admin'] = true;
 $wgGroupPermissions['fandom-editor']['fandom-admin'] = true;
+
+$wgGroupPermissions['staff']['exportuserdata'] = true;
```

Comment:
I think we should probably make this `util` rather than `staff` to be consistent with similar user data features being limited to `util` staff members.

Reply:
👍